datahub-classify/README.md

```diff
@@ -40,6 +40,7 @@ Infotype configuration is a dictionary with all infotypes at root level key. Eac
   2. Description
   3. Datatype
   4. Values
+- `ExcludeName` - optional exact match list for column names to exclude from classification for this info_type
```

Comment:
Any plan to support regex? Are we thinking to support it in future?

Reply:
good question- at this point, only if driven by customer requirement